npm2deb/templates.py

```diff
@@ -3,7 +3,6 @@
   * Initial release (Closes: #nnnn)
 
  -- %(debian_author)s  %(date)s
```

Comment:
isn't here still a newline ?

Reply:
That is a required blank line if you meant line 4 here.

Reply:
No, I meant after line 5

Reply:
Ending file with newline is ok, empty lines are not.

Before this change there was a line 6 in debian/changelog.

Reply:
got it.